### PR TITLE
Dashboard landing page

### PR DESCRIPTION
**Pull request by meg-gutshall**
# Description
Added dashboard functionality and fixed up breadcrumbs

# Type of change
* New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
Ran `bundle exec rspec`

# Screenshots
Screen Shot 2019-10-22 at 1 19 23 PM

### CODE REVIEW

**Comment by seanmarcia** [comment]
Hey @meg-gutshall this looks pretty good so far! I think, to make this more useable and to implement now we should:
- [ ] Remove the Diaper Bank Details, and Children Served Sections
- [ ] Remove all Lorem Ipsum Text
- [ ] In the Diaper Request History create a table displaying past requests.
  - [ ] Add links to create new Bulk and Family Requests (with the same permissions as currently being able to make those requests)
- [ ] In the Families Served section we should put some aggregate numbers in there:
  - [ ] Number of Families Served:
  - [ ] Number of Children Served:
  - [ ] Zip Codes Currently Served:
  - [ ] Break down of the number of kids served by size. (Optional)

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Prepares data to be shown to the users for their dashboard.
+class DashboardController < ApplicationController
+  respond_to :html, :js
+
+  def index
+    @partner = current_partner
+  end
+
+  # TODO: Remove commented code below
```
Comment (seanmarcia) [inline app/controllers/dashboard_controller.rb]: Can remove all this commented out code too

Comment (meg-gutshall) [reply app/controllers/dashboard_controller.rb]: Removed ✔️

**Comment by meg-gutshall** [comment]
Is currently failing five tests, most of which will be changed due to addition of the `ChildItemRequest` model. I need some help with passing the last failing test though.

Flash message `"Your account has been created successfully"` is supposed to appear upon `Partner` account registration. I don't think it came over correctly when I made the dashboard the new home page.

Breaking tests:
* `child_feature_spec` throws an error when trying to set a family variable: `let(:family) { create(:family, partner: partner) }` (line 20)
* `child_feature_spec` can't set a child variable from a `nil` family variable: `let!(:child) { create(:child, last_name: "Zeno", family: family) }` (line 42)
* `family_requests_feature_spec` throws and error on line 26: `child_item_requests = ChildItemRequest.count`
* `family_request_feature_spec` can't find the "Create New Family Diaper Request" link: `find('input[type="submit"]').click` (line 46)

**Comment by seanmarcia** [comment]
@mdworken Can you help get this over the finish line? :)

File: app/views/dashboard/_diaper_request_history.html.erb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            <th scope="col" class="col">Request Date</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          <% @partner_requests.each do |partner_request| %>
+            <tr class="row">
+              <td class="col-1"><%= link_to partner_request.id, partner_request_path(partner_request) %></th>
+              <td class="col-6"><%= partner_request.item_requests.map {|item| "#{item.quantity} of #{item.name}"}.join(", ") %></td>
+              <td class="col text-right"><%= link_to '<i class="fa fa-edit"></i>&nbsp;&nbsp;Pickup Sheet'.html_safe, family_request_pickup_sheets_path(partner_request), class: 'btn btn-sm btn-primary' %></td>
```
Comment (mlpinit) [inline app/views/dashboard/_diaper_request_history.html.erb]: We should probably put this behind a flipper feature flag as well.

File: app/views/dashboard/_diaper_request_history.html.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            <% end %>
+          </div>
+        </div>
+      </div>
+      <table class="table table-striped dashboard">
+        <thead>
+          <tr class="row">
+            <th scope="col" class="col-1">No.</th>
+            <th scope="col" class="col-6">Contents</th>
+            <th scope="col" class="col text-right">Edit Pickup Sheet</th>
```
Comment (mlpinit) [inline app/views/dashboard/_diaper_request_history.html.erb]: We should probably put this behind a flipper feature flag as well.

File: app/controllers/dashboard_controller.rb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Prepares data to be shown to the users for their dashboard.
+class DashboardController < ApplicationController
+  respond_to :html, :js
```
[the remark this answers is not in the document]
Comment (benreyn) [reply app/controllers/dashboard_controller.rb]: I dont think so, we can probably just remove this line completely.

**Pull request merged**